index.bs

```diff
@@ -6551,24 +6554,30 @@ engine.
 
     Concatenate |authenticatorData| and |clientDataHash| to form |attToBeSigned|.
 
+    Verify integrity of |certInfo|
+    - Verify that |x5c| is present.
+    - Verify that |aikCert| meets the requirements in [[#sctn-tpm-cert-requirements]].
+    - If |aikCert| contains an extension with OID `1.3.6.1.4.1.45724.1.1.4` (`id-fido-gen-ce-aaguid`) verify that the value of this
+        extension matches the <code>[=authData/attestedCredentialData/aaguid=]</code> in |authenticatorData|.
+    - Verify the |sig| is a valid signature over |certInfo| using the attestation public key in |aikCert| with the
+        algorithm specified in |alg|.
+
     Validate that |certInfo| is valid:
+    Note: |certInfo| is a TPMS_ATTEST structure.
     - Verify that `magic` is set to `TPM_GENERATED_VALUE`.
     - Verify that `type` is set to `TPM_ST_ATTEST_CERTIFY`.
     - Verify that `extraData` is set to the hash of |attToBeSigned| using the hash algorithm employed in "alg".
     - Verify that `attested` contains a `TPMS_CERTIFY_INFO` structure as specified in [[!TPMv2-Part2]] section 10.12.3,
-        whose `name` field contains a valid Name for |pubArea|,
-        as computed using the procedure specified in [[!TPMv2-Part1]]
-        section 16. Note that the hash algorithm is included within the attested `name` field of the TPMS_CERTIFY_INFO structure.
-    - Verify that |x5c| is present.
-    - Note that the remaining fields in the "Standard Attestation Structure" [[!TPMv2-Part1]]
+        whose `name` field contains a valid Name for |pubArea|, as computed using the procedure specified in [[!TPMv2-Part1]] section 16 using the nameAlg in the |pubArea|.
+
+        Note: The hash algorithm is also included within the attested `name`
+        field of the TPMS_CERTIFY_INFO structure and will also match nameAlg in |pubArea| when returned by the TPM.
+
+        Note: The remaining fields in the "Standard Attestation Structure" [[!TPMv2-Part1]]
         section 31.2, i.e., `qualifiedSigner`, `clockInfo` and `firmwareVersion` are ignored.
-        These fields MAY be used as an input to risk engines.
+        Depending on the properties of the |aikCert| key used, these fields may be obfuscated.
+        If valid, these MAY be used as an input to risk engines.
```

Comment:
The content of these fields is determined by which TPM hierarchy the AK is in.   (tl;dr: If the AK is in the Endorsement hierarchy the fields are valid, if the AK is in the Storage hierarchy, the fields are random - this is for privacy reasons). To provide information about where the AK is may be a privacy concern so this is not provided in the protocol. The properties of the AK (including whether to even trust it) are out of scope for this protocol and it would be up to the RP to know that out of band.